source/camera/CullingManager.h

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Copyright (C) 2011-2015  OpenDungeons Team
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CULLINGMANAGER_H_
+#define CULLINGMANAGER_H_
+#include "camera/CameraManager.h"
+#include "camera/SlopeWalk.h"
+
+#include "utils/Vector3i.h"
+
+#include <OgreRay.h>
+#include <OgrePlane.h>
+#include <set>
+
+class CameraManager;
+class GameMap;
+
+/*! \brief The CullingMangaer class is a class to effectivly 
```

Comment:
Some typos but overall, it helps understanding what you are trying to do. That's good.